dataflux_pytorch/benchmark/checkpointing/singlenode/train.py

```diff
@@ -132,26 +134,52 @@ def main():
     )
     trainer.fit(model, dataloader)
 
-    start = time.time()
-    for i in range(args.steps):
-        trainer.save_checkpoint(
-            os.path.join(args.ckpt_dir_path, f'ckpt_{i}.ckpt'))
-    end = time.time()
-    # command to clear kernel cache only works on MacOs and Linux.
-    if args.clear_kernel_cache and sys.platform in [
-            "darwin", "linux", "linux2", "linux3"
-    ]:
-        print("Clearing kernel cache...")
-        os.system("sync && sudo sysctl -w vm.drop_caches=3")
-    print("Average time to save one checkpoint: " +
-          str((end - start) / args.steps) + " seconds")
-    start = time.time()
-    for i in range(args.steps):
-        data = ckpt.load_checkpoint(
-            os.path.join(args.ckpt_dir_path, f'ckpt_{i}.ckpt'))
-    end = time.time()
-    print("Average time to load one checkpoint: " +
-          str((end - start) / args.steps) + " seconds")
+    if args.no_lightning:
+        # Create non-lightning checkpoint.
+        bucket_name, ckpt_path = path_utils.parse_gcs_path(args.ckpt_dir_path)
+        ckpt = dataflux_checkpoint.DatafluxCheckpoint(
+            project_name=args.project, bucket_name=bucket_name)
+        CKPT_PATH = ckpt_path if ckpt_path else "checkpoints/epoch0.ckpt"
```

Comment:
Should we add the step number to this file name so that it's not overwriting the same file repeatedly? There are GCS limits for how quickly you can rewrite the same object

Reply:
Good idea, will add.